mmv1/products/compute/api.yaml

```diff
@@ -3507,6 +3507,25 @@ objects:
         deprecation_message: This field is no longer in use, disk interfaces will be automatically determined on attachment. To resolve this issue, remove this field from your config.
         description: |
           Specifies the disk interface to use for attaching this disk, which is either SCSI or NVME. The default is SCSI.
+      - !ruby/object:Api::Type::String
+        name: 'sourceDisk'
+        description: |
+          The source disk used to create this disk. You can provide this as a partial or full URL to the resource.
+          For example, the following are valid values:
+
+          * https://www.googleapis.com/compute/v1/projects/project/zones/zone/disks/disk
+          * https://www.googleapis.com/compute/v1/projects/project/regions/region/disks/disk
+          * projects/project/zones/zone/disks/disk
+          * projects/project/regions/region/disks/disk
+          * zones/zone/disks/disk
+          * regions/region/disks/disk
```

Comment:
For better formatting, can we surround the variable segments with {}? like `projects/{project}/zones/{zone}/disks/{disk}` for both versions of the field

Reply:
Done.